packages/components/src/components/TabButton.vue

```diff
@@ -23,6 +23,13 @@ export default {
       required: true,
     },
     isActive: Boolean,
+    reference: String,
+  },
+
+  methods: {
+    clickTab() {
+      this.$root.$emit('clickTab', this.reference);
```

Comment:
Somewhere there's a `changeTab` event already being emitted.
https://github.com/getappmap/vscode-appland/blob/master/web/src/appmapView.js#L108-L114

Unless this is to delineate between the user clicking the tab explicitly vs selecting an object which is only represented in a single view?

Reply:
Yeah, sorry, I forgot to finish adding all of my notes to the PR description before I tagged you, but I wanted to delineate between clicking the tab and any other way of changing the view state.